autowebcompat/utils.py

```diff
@@ -195,3 +196,22 @@ def write_labels(labels, file_name='labels.csv'):
         writer.writerow(["Image Name", "Label"])
         for key, values in labels.items():
             writer.writerow([key, values])
+
+
+def read_boundary_boxes(file_name):
+    try:
+        with open(file_name, 'r') as f:
+            next(f)
+            reader = csv.reader(f)
+            boundary_boxes = {row[0]: ast.literal_eval(row[1]) for row in reader}
```

Comment:
Can we store the box with json? It should be more reliable

Reply:
I will add it into json file. Should I print it in the pretty format with `indent=4`? To keep it readable?

Reply:
No, let's put it in a single line, it should be readable anyway since it's just an array of arrays.